### DISCUSSION TITLE
Does mojo interoperate with existing C/C++ python extensions?

### DISCUSSION THREAD

**Discussion by nbecker** [modular/modular#364] [Q&A]
Title says it all.  Can I use my existing c++ python extensions from mojo?  That's a pretty strong requirement for me.

**Comment by lattner** [comment]
Mojo is currently good at talking to existing python packages, and this works whether they are hybrid internally or not.  We'd like to build out support for talking to C extensions directly but haven't built that yet.

**Reply by nbecker** [reply]
So the answer is, currently Mojo could call python code that could call code in C extensions, but Mojo cannot currently directly call code in C extensions?
